src/api/build.ts

```diff
@@ -30,21 +31,24 @@ export async function build({
 	output = 'src/node_modules/@sapper',
 	static: static_files = 'static',
 	dest = '__sapper__/build',
+	export_build = false,
 
 	bundler = undefined,
 	legacy = false,
 	ext = undefined,
 	oncompile = noop
 }: Opts = {}) {
-	bundler = validate_bundler(bundler);
 
+	const root = path.resolve('.');
 	cwd = path.resolve(cwd);
 	src = path.resolve(cwd, src);
```

Comment:
will this work? wouldn't it have to be relative to the root?

Reply:
I need a reference to the root from where the build is executed. cwd can be supplied as an absolute or relative path. And furthermore in build.ts cwd is reassigned to its absolute path with `cwd = path.resolve(cwd);`. After this there is no possibility to get the root back. Instead of `path.resolve('.')` maybe using `process.cwd()` would be more clear. I chose the former because it is more in line with the handling of the other paths. An other option would be to use `path.relative(cwd, '.')`, to get the needed reference and then get the root with `path.resolve(cwd, path.relative(cwd, '.'));`. But i think we are here only turning in circles.